Assets/Vimeo/Scripts/Config/VimeoVideo.cs

```diff
@@ -100,6 +100,30 @@ public int CompareTo(VimeoVideo other)
             return 0;
         }
 
+        public JSONNode GetMetadata()
+        {
+            try
+            {
+                JSONNode jsonObj = JSON.Parse(SeparateJSONObjectFromString(description));
+                return jsonObj;
+            }
+            catch (System.Exception e)
+            {
+                
+                throw new Exception("Could not read JSON file from the video description " + e);
+            }
+        }
+        
+        public string SeparateJSONObjectFromString(string content)
+        {
+            var matches = Regex.Matches(content, @"({.*})");
+            if (matches.Count == 0) {
+                return string.Empty;
+            }
+            string json = matches[0].Value.Replace("\\n", "");
```

Comment:
Not sure I understand why this is needed? I deleted these 2 lines and tests still pass.

Reply:
For some reason I thought this is necessary! just tested on my end and it works great without it